benchmarks/models.py

```diff
@@ -160,6 +160,9 @@ class Status:
     model_type = models.CharField(max_length=30, default='BaseModel')
     status = models.CharField(max_length=25, default='unknown')
 
+    # equivalent to ID until language changes were added: (ID 6756)
+    jenkins_id = models.IntegerField(default=999)
```

Comment:
```suggestion
    jenkins_id = models.IntegerField()
```
don't see why this needs a default?